app/script/properties/PropertiesType.js

```diff
@@ -27,6 +27,7 @@ z.properties.PROPERTIES_TYPE = {
     GOOGLE: 'contact_import.google',
     MACOS: 'contact_import.macos',
   },
+  EMOJI: 'settings.emoji.inline',
```

Comment:
`EMOJI` feels too generic, especially because more emoji settings will come in the future (like "tone selection")

Maybe change to something more specific, like `REPLACE_INLINE_EMOJI` and `settings.emoji.replace_inline` ?